src/test/kotlin/uk/gov/dluhc/printapi/service/PrintResponseFileServiceIntegrationTest.kt

```diff
@@ -0,0 +1,137 @@
+package uk.gov.dluhc.printapi.service
+
+import com.fasterxml.jackson.core.JsonParseException
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+import org.springframework.integration.support.MessageBuilder
+import org.springframework.messaging.MessagingException
+import uk.gov.dluhc.printapi.config.IntegrationTest
+import uk.gov.dluhc.printapi.config.SftpContainerConfiguration.Companion.PRINT_RESPONSE_DOWNLOAD_PATH
+import uk.gov.dluhc.printapi.jobs.ProcessPrintResponsesBatchJobIntegrationTest
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildBatchResponse
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildPrintResponse
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildPrintResponses
+import java.io.File
+import java.io.IOException
+
+internal class PrintResponseFileServiceIntegrationTest : IntegrationTest() {
+
+    @Nested
+    inner class FetchRemotePrintResponseFile {
+
+        @ParameterizedTest
+        @CsvSource(
+            value = [
+                "0, 0",
+                "0, 1",
+                "1, 0",
+                "1, 1"
+            ]
+        )
+        fun `should fetch remote file and unmarshall to a PrintResponses object`(
```

Comment:
Does the parameterized nature of this test add anything?
How is the scenario that has 1 BatchResponse and 1 PrintResponse different to any other combination?